pyproject.toml

```diff
@@ -21,6 +21,7 @@ requires = ["poetry-core>=1.0.0"]
 build-backend = "poetry.core.masonry.api"
 
 [tool.black]
+# 119 makes lines way too long to read.
 line-length = 100
```

Comment:
I guess I can accept 100 for this project, but a too short line length can cause unnecessary line breaks.

Reply:
100 I've found is just fine for that. If a line is so long it needs line breaks, it should probably be split up in other ways.